packages/wasm-utils/src/note/note.rs

```diff
@@ -402,4 +459,18 @@ mod test {
 		assert_eq!(note.group_id.to_string(), "0".to_string());
 		assert_eq!(note.curve.to_string(), "Curve25519".to_string());
 	}
+	#[test]
+	fn generate_note() {
+		let mut note_builder = NoteBuilder::default();
+		note_builder.backend = Backend::Arkworks;
+
+		note_builder.hash_function = HashFunction::Poseidon17;
+		note_builder.curve = Curve::Bn254;
+		note_builder.denomination = "17".to_string();
+		let note = note_builder.generate_note().unwrap();
+		assert_eq!(note.curve, Curve::Bn254);
+		assert_eq!(note.backend, Backend::Arkworks);
+		assert_eq!(note.denomination, "17".to_string());
```

Comment:
```suggestion
		assert_eq!(note.denomination, "18".to_string());
```
18 is more standard than 17 and should not be confused with the exponentiation value.